### PR TITLE
Use notify pubsub

### PR DESCRIPTION
**Pull request by insacuri**
# Motivation and Context

# What has changed

# How to test?

# Links

### CODE REVIEW

**Comment by ghost** [comment]
@insacuri, Spinnaker is deploying the PR to your environment

**Comment by ghost** [comment]
@insacuri, Spinnaker is deploying the PR to your environment

**Comment by ghost** [comment]
@insacuri, Spinnaker is deploying the PR to your environment

**Comment by ghost** [comment]
@insacuri, Spinnaker is deploying the PR to your environment

**Comment by ghost** [comment]
@insacuri, Spinnaker is deploying the PR to your environment

**Comment by ghost** [comment]
@insacuri, Spinnaker is deploying the PR to your environment

**Comment by ghost** [comment]
@insacuri, Spinnaker is deploying the PR to your environment

**Comment by ghost** [comment]
@insacuri, Spinnaker is deploying the PR to your environment

**Comment by ghost** [comment]
@insacuri, Spinnaker is deploying the PR to your environment

File: Pipfile [hunk 1]
```diff
@@ -13,6 +13,7 @@ flask-cors = "*"
 flask-httpauth = "*"
 gunicorn = "*"
 gevent = {version = "*", platform_python_implementation="=='CPython'"}
+google-cloud-pubsub = "*"
```
Comment (creativeappdevamit) [inline Pipfile:16]: should we use a version for this?

**Review by creativeappdevamit — CHANGES_REQUESTED** [review]

File: Pipfile (hunk 1 above)
Comment (insacuri) [reply Pipfile:16]: The version is locked in the Pipfile.lock.  Pipenv, by design, encourages you do leave things as * so when you regenerate the lockfile, you get minor package updates for 'free' (it's free until something breaks, but that's not as often as you'd think)

**Review by creativeappdevamit — APPROVED** [review]

**Pull request merged**